modules/transactionpool/transactionpool.go

```diff
@@ -80,28 +93,34 @@ func New(cs modules.ConsensusSet, g modules.Gateway) (*TransactionPool, error) {
 		knownObjects:        make(map[ObjectID]TransactionSetID),
 		transactionSets:     make(map[TransactionSetID][]types.Transaction),
 		transactionSetDiffs: make(map[TransactionSetID]modules.ConsensusChange),
+
+		persistDir: persistDir,
 	}
-	// Register RPCs
-	// TODO: rename RelayTransactionSet so that the conflicting RPC
-	// RelayTransaction calls v0.4.6 clients and earlier are ignored.
```

Comment:
technically, but there aren't many old nodes anymore. Ecosystem has done a good job of upgrading.